### ISSUE TITLE
what is is plan for Modules support with MSVC in Visual Studio 2019 version 16.8

### ISSUE THREAD

**Issue by mediabuff** [microsoft/cppwinrt#759]

**Comment by kennykerr** [comment]
We continue to make progress toward C++20 support. Most of the module support is already in C++/WinRT, but last I checked modules weren't quite ready as there were a few compatibility issues.

**Comment by mediabuff** [comment]
We should probably keep this open until there is full coverage of modules.

Specifically, the new Vc++ 16.8 has made lot of progress and IDE support. Has this been tested with cppwinrt ?

https://devblogs.microsoft.com/cppblog/standard-c20-modules-support-with-msvc-in-visual-studio-2019-version-16-8/

**Comment by kennykerr** [comment]
Yes I've been working with Cameron (the author of that post) and we've hit one or two new compiler bugs that need to be fixed before C++/WinRT will support modules. As there is no actionable work for C++/WinRT at this time I'm going to leave this issue closed.
